docs/contributing/pages/images.mdx

```diff
@@ -3,13 +3,13 @@ title: Images
 noindex: true
 ---
 
-If you want to add images to your docs, you can add them next to your `.mdx` files, under `./img` by convention.
+If you want to add images to your docs, you can add them in a `./img` folder under your `.mdx` files.
 You don't have to follow a rigid folder structure as long as the relative path (starts with `./`) to the image is correct.
 
-If an image is used in multiple places, it's best to put it in `includes/common-imgs`, reference it from there in a local `md[x]` file
+If an image is used in multiple places, it's best to put it in `includes/common-imgs`. You can reference it in a local `md[x]` file
 and then `Include` it in the target `mdx` files.
```

Comment:
I'm calling it an `include` file (rather than using the verb `Include`) to be consistent with other references in the contributing docs (https://docs.sentry.io/contributing/approach/sdk-docs/common_content/ & https://docs.sentry.io/contributing/approach/product-docs/write-index/). We can change this if you want to, but just want to make sure we're consistent.